doc(FAQ): guidance to layout 3 column content on 2x grid (#207)
carbon-design-system/carbon-website: src/pages/help/faq/index.mdx

Carbon charts are in development and classified as experimental. Check out the [experimental charts repo](https://carbon-design-system.github.io/carbon-charts/?experimental=true) to explore the work so far.

These charts will receive an updated color palette to match Carbon v10. Visual guidance and chart components are also in the works!

> ### How do I migrate an existing design from a 12 column grid to the new 16 column grid?

Reviewer: ```suggestion
### How do I migrate an existing design from a 12 column grid to the 16 column grid?
```

> There are many acceptable ways to move content from a 12 column grid onto a 16 column grid. If keeping the number of columns unchanged is not important to you, see below examples.

Reviewer: ```suggestion
There are many ways to move content from a 12 column grid onto a 16 column grid. If keeping the number of columns unchanged is not important to you, see below examples.
```

<Video vimeoId="356743483" />
<Video vimeoId="356743492" />
<Video vimeoId="356743508" />

> If keeping the number of columns to 3 is somehow significant to you, here are some other options.

Reviewer: ```suggestion
If keeping the number of columns to 3 is essential, here are some other options.
```

Author: Hahahah was I too snarky?
